notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -11655,6 +11655,50 @@ giant turtle [Gamera.]
 giant dinosaur [The Gyaos.]
 
 
+movie: Wing Commander (1999)
+============================
+
+:: Title
+Wing Commander
+
+:: Description
+The year is 2654, and it is up to a crop of fresh Terran Confederation
+military recruits to thwart the efforts of the cat-like Kilrathi aliens from
+exterminating the human race. It is loosely based on the video game series of
+the same name.
+
+:: Date
+1999-03-12
+
+:: References
+https://en.wikipedia.org/wiki/Wing_Commander_(film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+interstellar war [The Terran Confederation and the Kilrathi Empire were fighting a Milky way galaxy spanning war.]
+space colonization [A premise of the story is that the human race had reached out to colonize large swaths of the galaxy by the year 2654.]
+extraterrestrial being [The cat-like Kilrathi aliens were seeking the complete eradication of the human race.]
```

Comment:
https://www.themeontology.org/theme.php?name=felinoid%20being ?!

Reply:
Ah, that one is tossed in as minor on account that they were cat-like, but nothing was made of it to the extent where one wonders why they even bothered...